[WPE][cross-toolchain-helper] Update sysprof to 48.1

psaavedra opened the pull request:
#### 631cfa859332740fcffc3edae5c73e9532e667bb
<pre>
[WPE][cross-toolchain-helper] Update sysprof to 48.1
<a href="https://bugs.webkit.org/show_bug.cgi?id=298393">https://bugs.webkit.org/show_bug.cgi?id=298393</a>

Reviewed by Nikolas Zimmermann and Carlos Alberto Lopez Perez.

Add libdex and libpanel recipes and update sysprof patch
in meta-openembedded/meta-gnome layer:

* Update sysprof to 48.1
* libdex 0.10.1 with gobject-introspection enabled
* libpanel 1.6.0 with gtk4 and libadwaita support

This change also reverts the commit 283841@main that defaulted to
not preferring the system libsysprof-capture.

Reverts: <a href="https://bugs.webkit.org/show_bug.cgi?id=280184">https://bugs.webkit.org/show_bug.cgi?id=280184</a>

* Tools/yocto/meta-openembedded_and_meta-webkit.patch:
* Tools/yocto/targets.conf:

Canonical link: <a href="https://commits.webkit.org/299709@main">https://commits.webkit.org/299709@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/79169b1e0263b67e0abf98240c46f8691438e4af

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/119913 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/39606 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/30257 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/126231 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/71986 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/121789 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/40301 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/48183 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/91056 "Passed tests") | [✅ 🧪 win-tests](https://ews-build.webkit.org/#/builders/60/builds/60366 "Passed tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/122865 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/32182 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/107517 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/71613 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/133/builds/31212 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/25623 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/69874 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/101655 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/25813 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/129159 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/46833 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/35502 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/99679 "Passed tests") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/47199 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/103708 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/99524 "Passed tests") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/25271 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/44972 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/22985 "Passed tests") | [✅ 🛠 playstation](https://ews-build.webkit.org/#/builders/134/builds/43448 "Built successfully") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/46695 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/46161 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/49510 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/47847 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

nikolaszimmermann reviewed (approved):
Great thanks @psaavedra

psaavedra committed:
[WPE][cross-toolchain-helper] Update sysprof to 48.1
https://bugs.webkit.org/show_bug.cgi?id=298393

Reviewed by Nikolas Zimmermann and Carlos Alberto Lopez Perez.

Add libdex and libpanel recipes and update sysprof patch
in meta-openembedded/meta-gnome layer:

* Update sysprof to 48.1
* libdex 0.10.1 with gobject-introspection enabled
* libpanel 1.6.0 with gtk4 and libadwaita support

This change also reverts the commit 283841@main that defaulted to
not preferring the system libsysprof-capture.

Reverts: https://bugs.webkit.org/show_bug.cgi?id=280184

* Tools/yocto/meta-openembedded_and_meta-webkit.patch:
* Tools/yocto/targets.conf:

Canonical link: https://commits.webkit.org/299709@main
